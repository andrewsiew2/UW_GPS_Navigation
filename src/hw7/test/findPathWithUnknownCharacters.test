# Find a path with Dijkstra's algorithm when there are 2 possible paths
# with the same cost. It will take the shorter Path in terms of nodes

LoadGraph test testing.tsv

FindPath test GAMBLER DOTA
