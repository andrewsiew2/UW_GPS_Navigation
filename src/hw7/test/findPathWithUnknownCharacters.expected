# Find a path with Dijkstra's algorithm when there are 2 possible paths
# with the same cost. It will take the shorter Path in terms of nodes

loaded graph test

unknown character GAMBLER
unknown character DOTA